cnd/src/init_swap.rs

```diff
@@ -50,23 +51,37 @@ where
             let state = alice::State::accepted(request.clone(), *accept, seed);
             StateStore::insert(dependencies, id, state);
 
-            tokio::task::spawn(
-                create_swap::<D, alice::State<AL, BL, AA, BA, AI, BI>, AI, BI>(
-                    dependencies.clone(),
-                    accepted,
-                ),
-            );
+            tokio::task::spawn(create_alpha_watcher::<
+                D,
+                alice::State<AL, BL, AA, BA, AI, BI>,
+                AI,
+                BI,
+            >(dependencies.clone(), accepted.clone()));
+
+            tokio::task::spawn(create_beta_watcher::<
+                D,
+                alice::State<AL, BL, AA, BA, AI, BI>,
+                AI,
+                BI,
+            >(dependencies.clone(), accepted));
         }
         Role::Bob => {
             let state = bob::State::accepted(request.clone(), *accept, seed);
             StateStore::insert(dependencies, id, state);
 
-            tokio::task::spawn(
-                create_swap::<D, bob::State<AL, BL, AA, BA, AI, BI>, AI, BI>(
-                    dependencies.clone(),
-                    accepted,
-                ),
-            );
+            tokio::task::spawn(create_alpha_watcher::<
+                D,
+                alice::State<AL, BL, AA, BA, AI, BI>,
```

Comment:
hahaha, that's a fun merge mistake here :P 
Bob listening on Alice' state ;)

Fixing this...